stablehlo/reference/Ops.h

```diff
@@ -34,6 +34,11 @@ Tensor evalBatchNormInferenceOp(const Tensor &operand, const Tensor &scale,
                                 const Tensor &offset, const Tensor &mean,
                                 const Tensor &variance, APFloat epsilon,
                                 Axis featureIndex, ShapedType resultType);
+SmallVector<Tensor> evalBatchNormTraining(const Tensor &operand,
+                                          const Tensor &scale,
+                                          const Tensor &offset, APFloat epsilon,
+                                          const Axis featureIndex,
```

Comment:
I think the `const` part in `const Axis` is a bit over the top. Are we trying to guard against `evalBatchNormTraining` modifying the `featureIndex` parameter inside the body?